apps/sms/views/conversation/test/unit/information_test.js

```diff
@@ -366,8 +367,14 @@ suite('Information view', function() {
           sizeContent
         );
       }
-
-      sinon.assert.called(reportView.renderContactList);
+      if (receivers) {
+        sinon.assert.calledWithMatch(
+          reportView.renderContactList,
+          receivers.concat('sender')
+        );
+      } else {
+        sinon.assert.calledWith(reportView.renderContactList);
+      }
```

Comment:
proposal: instead of getting `receivers`, maybe have `expectedList` as parameter ? So that you can really use it in all tests, and call directly `sinon.assert.calledWithMatch(reportView.renderContactList, expectedList)` ?

What do you think ?